src/odpf/proton/metadata/resource/OptimusLog.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package odpf.proton.metadata.resource;
+
+option java_package = "io.odpf.proton.metadata.resource";
+option java_outer_classname = "OptimusLog";
+option go_package = "github.com/odpf/proton/metadata/resource";
+
+import "google/protobuf/timestamp.proto";
+
+message OptimusLogKey {
+  string urn = 1;
+}
+
+message OptimusLogMessage {
+  string urn = 1;
+  int32 version = 2;
+  string description = 3;
+  repeated OptimusConfig labels = 4;
```

Comment:
This is an incorrect representation of labels datatype although the message schema is the same.

Reply:
I've introduced following for it:
```proto
message OptimusJobLabel {
  string name = 1;
  string value = 2;
}
```